lib/crypto/base36.js

```diff
@@ -4,25 +4,31 @@
 
 'use strict'
 
-const crypto = require('crypto')
+const random = require('./random')
 
 function base36(len) {
   let out = []
-  while (out.length < len) {
-    let rand = crypto.randomBytes(len)
-    let randLen = rand.length
-    for (let i = 0; i < randLen; i++) {
-      let b = rand[i]
-      // 252-256 skews the base36 distribution, so skip those bytes
-      if (b < 252) {
-        out.push((b % 36).toString(36))
-        if (out.length === len) {
-          break
+  return random(len)
+    .then(bytes => {
```

Comment:
Apologies for the drive-by comment, but do we have any unit tests for this module? I don't see any in my local tree and it seems like this would be a great time to introduce some!